recipes/stillness-mode

```diff
@@ -0,0 +1 @@
+(ct :fetcher github :repo "neeasade/stillness-mode.el")
```

Comment:
The first element has to match the package name.

Reply:
frick, thanks, updated

Reply:
hmm - should the package file path be `stillness-mode.el` instead of `stillness.el`? (looking at the ci rn)

Reply:
looking through some other `-mode`'s on melpa, that appears to be the case - will edit.